packages/devtools_app/lib/src/connected_app.dart

```diff
@@ -109,16 +121,33 @@ class ConnectedApp {
 
   Future<void> initializeValues() async {
     await Future.wait([isFlutterApp, isProfileBuild, isDartWebApp]);
-    if (isFlutterAppNow) {
-      try {
-        final response = await serviceManager.flutterVersion;
-        _flutterVersion = FlutterVersion.parse(response.json);
-      } catch (e) {
-        logger.log(
-          'Failed to fetch flutter version from '
-          '`ConnectedApp.initializeValues`: $e',
-        );
-      }
+
+    // No need to check the flutter version for Flutter web apps, as the
+    // performance tools that consume [flutterVersionNow] are not available for
+    // flutter web apps.
+    if (isFlutterNativeAppNow) {
```

Comment:
Can you file a bug tracking adding support for flutterVersion for Flutter web apps? At some point it would be useful if that worked for flutter web so devtools extension authors don't have to worry about this..

Reply:
oddly enough I just checked with the gallery flutter web app and the flutterVersion service does register as expected. I don't see anything here that would prevent this from being registered on the web https://github.com/flutter/flutter/blob/5c09eecd0d5ea428d163de12d12d6bd9a7a8b16c/packages/flutter_tools/lib/src/vmservice.dart#L221.

Reply:
@grouma is this only broken in g3?

Reply:
Likely for now as Flutter Tools and Webdev have tighter restrictions on DevTools.